app/controllers/spree/admin/orders_controller_decorator.rb

```diff
@@ -27,6 +27,17 @@ def index
     # within the page then fetches the data it needs from Api::OrdersController
   end
 
+  def edit
+    @order.shipments.map &:refresh_rates
```

Comment:
there are two points about this method:
- there's a unless statement that we will need before the refresh shipping rates (out of scope for this PR)
https://github.com/spree/spree/blob/3c46d141c5cb2bedced63596178a0a79df3f0c9e/backend/app/controllers/spree/admin/orders_controller.rb#L54

- and also, the order advancement logic was removed in later versions of spree 2-0:
https://github.com/spree/spree/commit/8208b4bd0871b73be97bb2a7c7c1e0e4c04b3073#diff-1e2ba31309f1f1abd2a4b626036d316f

these are both out of scope of this PR.